.github/workflows/test-ci.yaml

```
@@ -23,19 +26,33 @@ jobs:
         with:
           poetry-version: 1.8.4
 
-      - name: Setup a local virtual environment (if no poetry.toml file)
+      - name: Setup a local virtual environment
         run: |
           poetry config virtualenvs.create true --local
           poetry config virtualenvs.in-project true --local
 
-      - name: Define a cache for the virtual environment based on the dependencies lock file
+      - name: Define a cache for the virtual environment
         uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)